frontend/src/components/drawer.jsx

```diff
@@ -146,99 +172,99 @@ export default function MiniDrawer() {
     .catch((error) => setLoggedin(!loggedin))
   };
 
-  if( loggedin ) {
-  return (
-    <Router>
-    <div className={classes.root}>
-      <CssBaseline />
-      <AppBar
-        position="fixed"
-        className={clsx(classes.appBar, {
-          [classes.appBarShift]: open,
-        })}
-      >
-        <Toolbar style={{backgroundColor:'black'}}>
-          <IconButton
-            color="inherit"
-            aria-label="open drawer"
-            onClick={handleDrawerOpen}
-            edge="start"
-            className={clsx(classes.menuButton, {
-              [classes.hide]: open,
+  if (loggedin) {
+    return (
+      <Router>
+        <div className={classes.root}>
+          <CssBaseline />
+          <AppBar
+            position="fixed"
+            className={clsx(classes.appBar, {
+              [classes.appBarShift]: open,
```

Comment:
For the react we should be using double space instead of full tab for the indentation